pkg/operator/ceph/object/user/controller.go

```diff
@@ -356,8 +381,8 @@ func generateStatusInfo(u *cephv1.CephObjectStoreUser) map[string]string {
 func (r *ReconcileObjectStoreUser) generateCephUserSecret(u *cephv1.CephObjectStoreUser) *corev1.Secret {
 	// Store the keys in a secret
 	secrets := map[string]string{
-		"AccessKey": *r.userConfig.AccessKey,
-		"SecretKey": *r.userConfig.SecretKey,
+		"AccessKey": r.userConfig.Keys[0].AccessKey,
```

Comment:
Could there ever be a userConfig with no keys, which would cause this to crash? Why is `Keys` a slice? Looks like we always expect one key pair.

Reply:
I don't think so, we would get an error before when fetching the user. The `Keys` is a slice because a user can have multiple keys, that's how both rgw admin ops API and `radosgw-admin` return the user definition.